### ISSUE TITLE
mio_dataset.py loading data from /opt

### ISSUE THREAD

**Issue by hashimSharif** [IBM/mini-era#10]
https://github.com/IBM/mini-era/blob/master/cv/CNN_MIO/mio_dataset.py#L8

The script loads data files from a system-specific path.

**Comment by jdwellman** [comment]
Good point; could maybe have some auto-configuration or something, but I'm not 100% sure  how best to address this.  This should probably address this in the README and either specify the location one should put the MIO dataset (to conform to above through that seems overly restrictive) or make the user edit the script, or set an environment variable (used in the script) that points to the MIO dataset (which I think IS listed as a requirement?).
